bin/client_package_debian

```diff
@@ -61,7 +62,7 @@ printf "Building Debian package '$PACKAGE_FILE' for architecture '$ARCH' ..\n" \
         --after-install bin/client_package/installation/post-install.sh \
         --after-remove bin/client_package/installation/post-uninstall.sh \
         -s dir -t deb \
-        build/client/mysterium_client=${OS_DIR_BIN}/ \
+        ${BINARY}=${OS_DIR_BIN}/ \
```

Comment:
# --license="MIT"
shouldn't it be GPLv3 ?

Reply:
Changed licence

Reply:
Removed maintainer